apps/greenhouse/index.html

```diff
@@ -0,0 +1,48 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+  ~ SPDX-License-Identifier: Apache-2.0
+-->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="src/assets/favicon.ico" sizes="any" />
```

Comment:
It would be a good idea to standardize the location for saving images across Heureka, Doop, Supernova and Example. We've introduced a src/img directory in each application and added all images with also both favicon-16x16.png and favicon-32x32.png files. These favicons have also been linked appropriately in the index.html file for consistency.